webapp/src/scriptmanager.tsx

```diff
@@ -451,6 +603,10 @@ export class ScriptManagerDialog extends data.Component<ScriptManagerDialogProps
                         </table>
                     </div>
                     : undefined}
+
+                {download && <sui.Modal header={lf("Preparing your zip file...")} isOpen={!!download} onClose={this.handleDownloadProgressClose}>
```

Comment:
Calling attention to this section, where I used sui.Modal instead of react-common Modal, since react-common Modal doesn't exist on the branch